docs/Changelog.md

```diff
@@ -10332,6 +10332,54 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain input and output types to integer tensors.</dd>
 </dl>
 
+### <a name="GatherND-11"></a>**GatherND-11**</a>
+
+  Given `data` tensor of rank r >= 1, and `indices` tensor of rank q >= 1, gathers
+  slices of `data` into an output tensor of rank q + r - indices_shape[-1] - 1.
```

Comment:
Please describe the indexing mechanism to compute output values. It'd be very helpful if we have something like
```
output[i][j] = data[indices[i]][indices[j], :]
```

Reply:
It is very hard to write a single equation for this op - in fact the equation written by tf gather_nd is not even clear to me. Instead, I have a detailed section on how to compute the output values in my latest commit and I have linked the explanation to an illustrative example. I believe this drives home the usage of the operator better than some equation that may be hard to parse.